### PR TITLE
removed two add tag labels on add tag area on media uploader

### PR DESCRIPTION
**Pull request by rnagulapalle**
[[Media][Add Video] Tag area has two "add tag" labels.]
(https://app.asana.com/0/348819471518170/878313396719388/f)

### CODE REVIEW

File: components/mediaUploader.css [hunk 1]
```diff
@@ -123,3 +123,6 @@
   text-align: center;
   opacity: 1;
 }
+.md-block.md-icon-float.md-input-has-placeholder > label{
+  display:none!important;
```
Comment (alexgurrola) [inline components/mediaUploader.css]: We don't use `!important` in css.  This also has merge conflicts with the master branch.

File: components/mediaUploader.css [hunk 2]
```diff
@@ -264,6 +264,8 @@ md-dialog {
   text-align: center;
   opacity: 1;
 }
+.md-block.md-icon-float.md-input-has-placeholder > label{
```
Comment (alexgurrola) [inline components/mediaUploader.css]: This needs to be a little more specific for your use case, or this `display:none` will bleed into other components.

File: components/mediaLibrary.html [hunk 3]
```diff
@@ -178,7 +178,9 @@
 
                             <!--
                             TODO: Discuss merits of lazy loading vs. background image
-                            <img ng-if="media.data.mime.indexOf('image') > -1" stratus-src="{{media.data.url || media.data.file}}">-->
+                            <img ng-if="media.data.mime.indexOf('image') > -1" data-plugin="lazy"
```
Comment (alexgurrola) [inline components/mediaLibrary.html]: This is rolling back a component to an unsupported version and is also commented out.  This really doesn't make sense.

Comment (rnagulapalle) [reply components/mediaLibrary.html]: Thats my bad.. i suppose to commit mediaUpload.html but somehow it did not go through. fixed and squashed...

**Pull request merged**